cmd/kube-apiserver/app/aggregator.go

```diff
@@ -225,6 +225,7 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	{Group: "storage.k8s.io", Version: "v1beta1"}:                {group: 16800, version: 9},
 	{Group: "apiextensions.k8s.io", Version: "v1beta1"}:          {group: 16700, version: 9},
 	{Group: "admissionregistration.k8s.io", Version: "v1alpha1"}: {group: 16700, version: 9},
+	{Group: "scheduling.k8s.io", Version: "v1alpha1"}:            {group: 16600, version: 9},
```

Comment:
Should the version be 9? I don't know what it should be. I just wanted to make sure it is the right number.

Reply:
```
// VersionPriority controls the ordering of this API version inside of its group.  Must be greater than zero.
// The primary sort is based on VersionPriority, ordered highest to lowest (20 before 10).
// The secondary sort is based on the alphabetical comparison of the name of the object.  (v1.bar before v1.foo)
// Since it's inside of a group, the number can be small, probably in the 10s.
VersionPriority int32
```
from the code comment, we can see that the version is set by human and the version number is as same `v1alpha1`  which is set before

Reply:
Yeah, I see all sort of versions, like v1, v2alpha1, v1alpha1 set to 9, but from the comment, I guess it doesn't matter and the version applies to the items in the same API group. So, we should be fine.